### DISCUSSION TITLE
Companies using Kamal in production

### DISCUSSION THREAD

**Discussion by nickhammond** [basecamp/kamal#683] [Ideas]
Hey all,

As Kamal grows it'd be great to help highlight some of the companies that are utilizing Kamal in production. A validation and trust message that helps communicate to people that Kamal is a great fit for a lot of companies.

Where can we highlight these companies? Maybe something we can throw on the marketing site with a handful of those companies?

**Comment by igor-alexandrov** [comment]
I will start, we at https://jetrockets.com adopted Kamal at the very early stage of the project (when it was MRSK). Within 2023 we switched almost all our projects to Kamal and happy with this.

**Comment by etmartinkazoo** [comment]
We have a number of large, business critical, apps in production with Kamal at [Brem](https://brem.io). They are both doing a lot of traffic daily, running on Sqlite and using Solid Queue for background jobs.
